libkmod/libkmod-internal.h

```diff
@@ -68,51 +70,86 @@ enum kmod_file_compression_type {
 	KMOD_FILE_COMPRESSION_ZLIB,
 };
 
-struct kmod_list *kmod_list_append(struct kmod_list *list, const void *data) _must_check_ __attribute__((nonnull(2)));
-struct kmod_list *kmod_list_prepend(struct kmod_list *list, const void *data) _must_check_ __attribute__((nonnull(2)));
+struct kmod_list *kmod_list_append(struct kmod_list *list, const void *data) _must_check_
+__attribute__((nonnull(2)));
+struct kmod_list *kmod_list_prepend(struct kmod_list *list, const void *data) _must_check_
+__attribute__((nonnull(2)));
 struct kmod_list *kmod_list_remove(struct kmod_list *list) _must_check_;
-struct kmod_list *kmod_list_remove_data(struct kmod_list *list,
-					const void *data) _must_check_ __attribute__((nonnull(2)));
-struct kmod_list *kmod_list_remove_n_latest(struct kmod_list *list,
-						unsigned int n) _must_check_;
-struct kmod_list *kmod_list_insert_after(struct kmod_list *list, const void *data) __attribute__((nonnull(2)));
-struct kmod_list *kmod_list_insert_before(struct kmod_list *list, const void *data) __attribute__((nonnull(2)));
-struct kmod_list *kmod_list_append_list(struct kmod_list *list1, struct kmod_list *list2) _must_check_;
+struct kmod_list *kmod_list_remove_data(struct kmod_list *list, const void *data)
+	_must_check_ __attribute__((nonnull(2)));
+struct kmod_list *kmod_list_remove_n_latest(struct kmod_list *list, unsigned int n)
+	_must_check_;
+struct kmod_list *kmod_list_insert_after(struct kmod_list *list, const void *data)
+	__attribute__((nonnull(2)));
+struct kmod_list *kmod_list_insert_before(struct kmod_list *list, const void *data)
+	__attribute__((nonnull(2)));
+struct kmod_list *kmod_list_append_list(struct kmod_list *list1, struct kmod_list *list2)
+	_must_check_;
```

Comment:
what's "lhs suggestion?

Reply:
ahh... lhs == left hand side?

Linux kernel has coding style guideline for that, although I don't really like it much since it changes the order between declaration and definition (but afair it's exactly because the syntax would otherwise not be accepted by compilers). See https://www.kernel.org/doc/html/latest/process/coding-style.html#function-prototypes